README.md

```diff
@@ -84,16 +84,23 @@ To start launcher use
 ```bash
 # JUPYTERHUB_URL is URL where JupyterHub is running. If path like `/jupyter` then origin header is appended.
 export JUPYTERHUB_URL=http://172.17.0.1:8000
+# JWT_SECRET is secret with which JWT tokens are encoded/decoded
+export JWT_SECRET=$(openssl rand -hex 32)
+# Root directory where read only forcings can be found
+export FORCING_ROOT_DIR=/mnt/data/forcing
 gunicorn -w 4 -b 0.0.0.0:8888 ewatercycle_experiment_launcher.serve:app
 ```
 
 Goto http://localhost:8888/ui/ for Swagger UI.
 
 The JupyterHub and Experiment Launcher both use local OS accounts for authentication and authorization.
+To generate a notebook you need to
+1. POST to /auth with Basic authentication to receive a JWT token
+2. POST to a notebook path, like /hello, with `Authorization: Bearer <jwt token>` header.
```

Comment:
Getting 401 unauthorized for everything I try (WSL problem?)

Reply:
Hmm, JH uses the OS user accounts to authenticated against. Which works on my linux box. Not sure how accounts work in WSL. Mabye add user with `sudo useradd someone;sudo passwd someone`.

Reply:
Still no luck